src/tests/tests/first_parties.js

```diff
@@ -0,0 +1,32 @@
+(function() {
```

Comment:
Sorry, I meant QUnit module, to help us organize our tests and also so that we can use stuff like 'beforeEach'.